lib/persisted-model.js

```diff
@@ -549,6 +574,21 @@ module.exports = function(registry) {
     var PersistedModel = this;
     var typeName = PersistedModel.modelName;
     var options = PersistedModel.settings;
+    // This is the defualt verb used for 2.X
+    var configurableVerb = {
+      updateAttributes: 'put',
+      updateOrCreate: 'put',
+      replaceOrCreate: 'patch',
+      replaceById: 'patch',
```

Comment:
As I commented earlier, I find it weird for `PATCH` to make a full replace. It also makes the surface of the breaking change between 2.x and 3.x bigger.